content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,19 +2,19 @@
 Multiple sales teams
 ====================
 
-Odoo lets you manage several sales teams, departments, or channels, each with their own unique
-sales processes, using *Sales Teams*.
+Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
+their own unique sales processes.
 
 
 Create a new Sales Team
```

Comment:
Would use [sentence case](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings) here, just like the H1.
```suggestion
Create a new sales team
```